enterprise/server/remote_execution/blockio/blockio.go

```diff
@@ -363,9 +364,25 @@ func (c *COWStore) ReadAt(p []byte, off int64) (int, error) {
 				p[i] = 0
 			}
 		} else {
-			if _, err := readFullAt(chunk, p[:readSize], chunkRelativeOffset); err != nil {
+			s, err := chunk.SizeBytes()
```

Comment:
Could you rename SizeBytes to something like DataSize? The different types of `sizes` is a bit confusing